docs/source/models/gbd2017/causes/latent_tuberculosis/index.rst

```diff
@@ -4,6 +4,111 @@
 Latent Tuberculosis
 ===================
 
-.. todo:
+Validation and Verification Criteria
+------------------------------------
 
-   all of this
+Obejective
+++++++++++
+External
+ - Model results should be checked against local TB epidemiology
+    - e.g. rate of decline in burden of disease should be compared
+      with historical evidence.
+ - Compare our results (e.g. ICERs) to similar models or empirical
+   assessments where possible
+Internal
+ - Test hypotheses without develop a full simulation. 
+ - Calibrate simulation baseline against GBD 2017 results
+    - Sim outputs mean should perfectly match GBD results.
+    - Sim outputs uncertainty should be wider than GBD results,
+      because both stochastic and parametric uncertainty are included.
+
+V&V strategy
+++++++++++++
+Model Validation
+ - Check the logical structure and input data for concept model,
+   make sure
+    - the theories and assumptions underlying the conceptual model are correct.
+    - the data to build, evaluate, and test model are correct.
+Model Verification
+ - Check the translation from concept model to script based on test data,
+   make sure
+    - the computer programming and implementation of the concept model is correct.
+    - the output of the model can be calibrated against GBD results.
+
+logic
++++++
+- Parent cause is the sum of child causes
+    - Fatal: Deaths (CSMR, Excess MR), YLLs
+    - Non-fatal: Cases, YLDs, Prevalence, Incidence
+    - DALYs = YLLs + YLDs
+- By location-/age-/sex-
+- Start from count space
+- We expect that total active TB cases less than LTBI cases for sim output
+  of event count
+
+How GBD post-processing results
++++++++++++++++++++++++++++++++
+GBD starts from All-form TB results
+    1. Find proportion of HIV+ cases among all TB cases
+    2. Disaggregate into HIV+ TB and HIV- TB
+    3. Find proportion of drug-resistant cases among HIV+ TB cases
+       and HIV- TB cases
+    4. Disaggregate into:
+        - drug-susceptible TB, multidrug-resistant TB, and extensively
+          drug-resistant TB
+        - HIV+ drug-susceptible TB, HIV+ multidrug-resistant TB,
+          and HIV+ extensively drug-resistant TB
+
+Formula
++++++++
+For certain location-/age-/sex-
+    - Deaths due to all causes equal to sum of:
+        - Deaths due to all-form TB (aggregate all child active TB causes)
+        - Deaths due to HIV resulting in other diseases
+        - Deaths due to other causes
+Apply the formula to other measures (e.g. DALYs)
+
+Steps of model verification
```

Comment:
I think it would be useful to clarify what is done automatically in vivarium post-processing by the software engineers and what should be done by the research folks. I'm not sure what the answer to this is, but I think it would be good to include here.